stories/shared-components/selected-species/Lozenge.stories.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import SpeciesLozenge from 'src/shared/components/selected-species/SpeciesLozenge';
+
+import speciesData from './speciesData';
+
+import { humanGenome } from 'src/shared/components/selected-species/fixtures/speciesTestData';
+
+import styles from './SelectedSpecies.stories.module.css';
+
+export default {
+  title: 'Components/Shared Components/Selected Species'
+};
+
+export const SpeciesLozengeStory = () => {
+  const species = speciesData[0];
+  return (
+    <>
```

Comment:
The usage of react fragment seems not needed here as everything is contained within the `div` below

Reply:
✅